nebula-logger/main/logger-engine/classes/LogEntryEventBuilder.cls

```diff
@@ -423,6 +599,10 @@ global with sharing class LogEntryEventBuilder {
     }
 
     private String getSObjectClassification(Schema.SObjectType sobjectType) {
+        if (sobjectType == null) {
```

Comment:
safe navigation here can cut down on the complexity here and for the next addition (~ line 639) by reducing the need for these if statements!

Reply:
I thought about using safe navigation, but using a guard clause at the top cuts out the rest of the methods logic from running. If I switch to using safe navigation (and remove the guard clause), then the full method executes, which eats up some CPU usage.